packages/graphql/src/contracts.js

```diff
@@ -275,18 +275,24 @@ export function setNetwork(net, customConfig) {
     )
   }
 
-  let marketplaceVersion
+  let largestMarketVersion
```

Comment:
nit: I'd suggest calling this "latestMarketplaceVersion"

Reply:
agreed 👍